src/mem/old_api.cc

```diff
@@ -585,14 +585,12 @@ memFreeBufFunc(size_t size)
     }
 }
 
-/* MemPoolMeter */
-
 void
-Mem::PoolReport(const MemPoolStats * mp_st, const MemPoolMeter * AllMeter, std::ostream &stream)
+Mem::PoolReport(const MemPoolStats * mp_st, const Mem::PoolMeter * AllMeter, std::ostream &stream)
```

Comment:
The two here removed in [0988026](https://github.com/squid-cache/squid/pull/1183/commits/098802604003c4e1cd11f466c68973a9625d1997). I'm not sure what others you are seeing that qualifies as "quite a few" - note that the similar uses in mem/Pool.h are all in global scope classes/methods.

Reply:
> I'm not sure what others you are seeing

Three more were removed under https://github.com/squid-cache/squid/pull/1183#discussion_r1026923903. I see only one left in the current PR code:

```C++
//Mem::PoolMeter const &Mem::AllocatorProxy::getMeter() const STUB_RETSTATREF(Mem::PoolMeter)
```

> note that the similar uses in mem/Pool.h are all in global scope classes/methods.

Noted. I probably misidentified some of them. Sorry.

To avoid changing many of those lines twice, please use `using Mem::PoolMeter` or similar statements in class declarations for classes that you are working on moving into the Mem namespace, especially classes that already have open PRs implementing that move. It would be much easier to find and remove excessive `using Mem::PoolMeter` statements (during class moves) than to correctly remove unnecessary `Mem::` prefixes in moved code.

Reply:
Done in [c4090a4](https://github.com/squid-cache/squid/pull/1183/commits/c4090a4751fe17cd068db6da605da4b32d48d895).